tests/test_opds.py

```diff
@@ -762,9 +762,10 @@ def test_page_feed(self):
         pagination = Pagination(size=1)
 
         def make_page(pagination):
+            SessionManager.refresh_materialized_views(self._db)
```

Comment:
...and this was a mistake because it broke the tests. I think because the materialized view doesn't get wiped at the end of a test? I'm not sure, but I reverted the code.